packages/account/src/Types/common-prop.types.ts

```diff
@@ -0,0 +1,14 @@
+export type TFile = {
```

Comment:
just to clarify:
why do we need these two separate files? 
common-prop.type.ts and common-prop.types.ts